runtime/lua/man.lua

```diff
@@ -729,6 +728,8 @@ function M.open_page(count, smods, args)
 
   if not ok then
     error(ret)
+  elseif vim.fn.exists('g:pager') == 1 then
+    set_options(vim.g.pager)
```

Comment:
```suggestion
```

Reply:
And change the line below from `set_options(false)` to `set_options()`

Reply:
Sorry forgot this one